pkg/process/metrics.go

```diff
@@ -10,7 +10,7 @@ import (
 	"path/filepath"
 
 	"github.com/zeebo/admission/admproto"
-	"gopkg.in/spacemonkeygo/monkit.v2"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
```

Comment:
I guess this is another gofmt change. I am not sure if it causes any trouble, but it would be better to be on the safe side and remove it from this PR.